compiler-rt/lib/sanitizer_common/sanitizer_platform_limits_posix.cpp

```diff
@@ -545,6 +545,10 @@ unsigned struct_ElfW_Phdr_sz = sizeof(Elf_Phdr);
   unsigned struct_sioc_vif_req_sz = sizeof(struct sioc_vif_req);
 #endif
 
+#  if !SANITIZER_FREEBSD && !SANITIZER_NETBSD
```

Comment:
This is the same as the freebsd and netbsd one in sanitizer_platform_limits_[free|net]bsd.cpp

Can't we just do it in here only?